### DISCUSSION TITLE
Numbering for headings...

### DISCUSSION THREAD

**Discussion by BruceCaron** [knowledgefutures/pubpub#1219] [Ideas]
A reader requested that I use numbering for the headings/subheadings of <https://openscientist.pubpub.org/pub/play/release/1>.   So I'm passing this on as a feature request.  I know you do numbering for images and equations.

**Comment by gabestein** [comment]
Thanks Bruce! Very cool idea. I was thinking we could also potentially combine this with figure headings so that you could have, for example, figure 2.1, 2.2, etc. if you turn on numbered headings.
